packages/jwt/README.md

```diff
@@ -0,0 +1,32 @@
+# `@byu-oit/jwt`
+
+> Provides helpful functions to retrieve a specified BYU .well-known URL and verify BYU signed JWTs.
+
+## Usage
+
+```javascript
+import {ByuJwt} from '@byu-oit/jwt'
+
+const byuJwt = new ByuJwt({ issuer: 'https://api.byu.edu' })
+
+const jwt = byuJwt.verify('[your jwt]')
+
+/** Access the jwt payload information */
+const { byuId } = jwt.payload
+
+/** Access the jwt header information */
+const { alg } = jwt.header
+```
+
+> **Note**
+> Please refer to the API documentation if you need to see what information is made available in the jwt payload or header.
+
+## Options
+
+| property        | type    | default            | description                                                                                                                                                                                                                                                   |
+|-----------------|---------|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| basePath        | string  | `undefined`        | will validate that the audience starts with the provided basePath in production.                                                                                                                                                                              |
+| cacheDuration   | number  | `60 * 60` (1 hour) | the default cache duration in seconds if one isn't provided by the max-age header for the certs or openid configuration                                                                                                                                       |
```

Comment:
To be clear, we're checking the [`Cache-Control` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) and at BYU, we only happen to use the `max-age` directive in that header.

Reply:
✅